src/Func.cc

```diff
@@ -128,6 +128,22 @@ void Func::AddBody(detail::StmtPtr /* new_body */,
 	Internal("Func::AddBody called");
 	}
 
+void Func::AddBody(const zeek::detail::function_ingredients* ingredients)
+	{
+	Internal("Func::AddBody called");
+	}
+
+void Func::UpdateBodies()
+	{
+	static auto group_disabled = [](const auto& g)
+	{
+		return g->IsDisabled();
+	};
+
+	for ( auto& b : bodies )
+		b.disabled_groups = std::count_if(b.groups.cbegin(), b.groups.cend(), group_disabled);
```

Comment:
Matter of taste but I'd just compress this inline:
```suggestion
		b.disabled_groups = std::count_if(b.groups.cbegin(), b.groups.cend(), [](const auto& g) { return g->IsDisabled(); });
```

Reply:
I don't know why, but seems clang-format gets very confused around tabs and spaces if I do that. I might stick with the longer version for now.

![Screenshot-20221021-182458](https://user-images.githubusercontent.com/2100509/197243596-0878b238-273f-4f77-a671-5fe48f98513e.png)